scripts/scr_initialize_custom/scr_initialize_custom.gml

```diff
@@ -1872,7 +1876,7 @@ function scr_initialize_custom() {
 			}],
 		]
 	};
-	if (global.chapter_name == "Salamanders") or (array_contains(obj_creation.adv, "Crafters")) { //salamanders squads
+	if (global.chapter_name == "Salamanders") or (scr_has_adv ("Crafters") and ("Assault Docrine")) { //salamanders squads
```

Comment:
I think here is a typo and you forgot scr_has_adv for assault doctrine.

I'm not sure what's the idea here and why you need both crafters and assault doctrine, but okay.

Reply:
It hasn't been changing anything for Salamander assault squads for a while the change I did was on a whim.